challenges/test/lib/src/test/java/code/stacksandqueuestest/PseudoQueueTest.java

```diff
@@ -0,0 +1,34 @@
+package code.stacksandqueuestest;
+
+import code.stacksandqueues.PseudoQueue;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class PseudoQueueTest {
```

Comment:
You wrote out an exception on the dequeue method, don't forget to test it! What happens if I try to dequeue when everything is empty? Does the exception work or do we get an error?